packages/build-utils/src/convert-runtime-to-plugin.ts

```diff
@@ -124,22 +131,17 @@ async function readJson(filePath: string): Promise<{ [key: string]: any }> {
   }
 }
 
-async function readVercelConfig(
-  workPath: string
-): Promise<{ functions?: BuilderFunctions; regions?: string[] }> {
-  const vercelJsonPath = join(workPath, 'vercel.json');
-  return readJson(vercelJsonPath);
-}
-
 /**
  * If `.output/functions-manifest.json` exists, append to the pages
  * property. Otherwise write a new file. This will also read `vercel.json`
  * and apply relevant `functions` property config.
  */
 export async function updateFunctionsManifest({
+  vercelConfig,
   workPath,
   pages,
 }: {
+  vercelConfig: { functions?: BuilderFunctions; regions?: string[] };
```

Comment:
Same here, can we use the actual type, or should it be a just a subset?

Reply:
We don't need all the props, just these two are used in the conversion function here.

Also I don't think we can depend on `@vercel/client` from `@vercel/build-utils` since build-utils builds before client so that would make a cyclical dependency

Reply:
I think in the future, we could try to merge `routing-utils` and `build-utils` into a single package and then move the VercelConfig type there instead of `client`

Reply:
Makes sense. However, we still need the `trailingSlash` and `cleanUrls` property to ensure we can output the correct routes for those settings.

Reply:
Unless you want to add that in a follow-up PR?